frameworks/remix-run/application/src/routes/$langstore._index.tsx

```diff
@@ -46,6 +47,6 @@ export const loader: LoaderFunction = async ({ request, params }) => {
 };
 
 export default () => {
-    const { data } = useLoaderData();
+    const { data } = useLoaderData() as { data: LandingPageType };
```

Comment:
`useLoaderData<typeof loader>();` + change the loader using `LoaderFunctionArgs`